### PR TITLE
Add link to userguide to enable k8s aggregation layer

### PR DESCRIPTION
**Pull request by font**
/sig multicluster

### CODE REVIEW

**Comment by perotinus** [comment]
@font I'll LGTM this after #174 goes in. Thanks, this is a good change!

File: docs/userguide.md [hunk 1]
```diff
@@ -53,7 +53,13 @@ $
...
 ### Aggregated API server
 
-You can deploy the cluster registry as an aggregated API server like so:
+Before deploying the cluster registry as an aggregated API server, take a look at
+https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#enable-apiserver-flags
+which talks about what `kube-apiserver` flags need to be enabled in order to
+enable the aggregation layer.
+
+Once the `kube-apiserver` aggregation layer is enabled, you can deploy the
+cluster registry as an aggregated API server like so:
```
Comment (perotinus) [inline docs/userguide.md]: Actually, looking at the context in #172, maybe we should add a section about the names specifically? Or else find a link to a site that explains it in more detail? The link here gives a brief overview of the flags, but doesn't necessarily explain how they should be used with a custom CA.

Comment (font) [reply docs/userguide.md]: Unfortunately, there is lack of proper documentation on this topic. The link to the list of flags doesn't have much detail on how values of those flags can vary in different configurations, but AFAICT it's all we have.

I also thought about documenting it some myself, but I'm not 100% certain what all the security ramifications are for specifying multiple or removing the `--requestheader-allowed-names` flag. So I figured I should avoid providing any potentially misleading information in our docs that others may attempt to use in a production deployment of the CR. Especially when the k8s docs themselves do not document it.

Thoughts?

Comment (perotinus) [reply docs/userguide.md]: That seems reasonable to me: if the k8s docs don't discuss this, it's better for us not to provide info.

We can revisit this, though, if someone else brings up the same issue.

**Pull request merged**